_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -0,0 +1,45 @@
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "rust")
+
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing. Of course, we also had mentor Carol Willing (past Director of the Python Software Foundation) returning from HackIIllinois 2017 to work on [Jupyter](https://github.com/jupyter/jupyter).
+
+# Projects
+
+Tons of Open Source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large Open Source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These influential projects gave students and mentors the opportunity to work together to contribute to some of the most common tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing Open Source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn).  These Open Source projects have immense opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to only contributing to Open Source projects, though. Many students developed their own Open Source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul), even has 70+ stars on GitHub as of the time of publication, and was [mentioned in a talk at Google I/O.](https://youtu.be/lhZOFUY1weo?t=19m5s) And it was born right here at HackIllinois.
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/open-source-hackillinois-2018/puppeteer.jpg "puppeteer")
```

Comment:
Change caption to "The Puppeteer to Istanbul team presenting at closing ceremonies"